.pylintrc

```diff
@@ -0,0 +1,5 @@
+[MAIN]
+
+# Files or directories to be skipped. They should be base names, not
+# paths.
+ignore=_version.py
```

Comment:
Is the `_version.py` file autogenerated?

Reply:
Yep. `versioneer` generates it.